### PR TITLE
QuantifiedCode_issues

### PR DESCRIPTION
**Pull request by the-zebulan**
Issue: #929

* Fixed "non-Pythonic" method name
* Converted to set comprehension
* Used `is None` instead of `== None`
* Fixed "non-Pythonic" variable names
* Used an iterator instead of list comprehension inside of `any`
* Removed `None` as redundant second argument to `dict.get()`
* Merged consecutive if-statements (also used `elif`'s instead of multiple `if`'s)

### CODE REVIEW

File: server/models.py [hunk 1]
```diff
@@ -755,14 +765,11 @@ class Backup(Model):
     def can(cls, obj, user, action):
         if action == "create":
             return user.is_authenticated
-        if not obj:
+        elif not obj:
             return False
-        if user.is_admin:
+        elif user.is_admin or (action == "view" and user.id in obj.owners()):
```
Comment (Sumukh) [inline server/models.py]: Do you think it makes sense to break the second part of this conditional into it's own `elif` branch? If we think of each conditional representing a specific case - the case of something being an admin is very different from being an owner of the object. 

I think quantifiedcode will be just as happy with 
```
elif user.is_admin
   return True 
elif action == "view"  and user.id in obj.owners():
   return True
```

**Comment by the-zebulan** [comment]
Uh oh, tests failed?

**Comment by Sumukh** [comment]
I think the test failure is unrelated - it's because we haven't mocked the autograder. I'll look into it and try to rerun the test.

**Comment by the-zebulan** [comment]
Ok, just let me know if I messed something up. If so, I will try and fix it ASAP.

**Comment by Sumukh** [comment]
Github PR tool: In the future if you include the phrase `Resolves #929` in your first comment - the issue will automatically be closed when the PR is merged :) 

https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by the-zebulan** [comment]
Thanks for the tip!

I didn't know typing that would close the issue when the PR was merged. I'll make sure to use that message in the future.

**Pull request merged**